main.py

```diff
@@ -148,16 +148,21 @@ def setup_conversion_settings(self):
         self.conversion_settings = {setting: tk.BooleanVar(value=True) for setting in settings}
         self.conversion_settings["notes"].set(False)
         self.conversion_settings["objects"].set(False)
+
+        self.gm_platform_settings = "windows"
```

Comment:
It would be nice on this line to have a default value get changed, so if I am running this program on windows, this would be by default "windows", but if I am running on linux, then it would be "linux", because I assume on linux you have to change the setting to be linux. We can use the platform module to detect the OS running GM2Godot, or platform.system() similarly how it's used in icon.py.